src/app/client/src/app/modules/core/components/main-footer/main-footer.component.ts

```diff
@@ -67,26 +75,32 @@ footerAlign() {
 
 
   redirectToDikshaApp() {
-    let applink = this.configService.appConfig.UrlLinks.downloadDikshaApp;
-    const sendUtmParams = _.get(this.activatedRoute, 'firstChild.firstChild.snapshot.data.sendUtmParams');
-    const utm_source = this.userService.slug ? `${this.instance}-${this.userService.slug}` : this.instance;
-    if (sendUtmParams) {
-      observableCombineLatest(this.activatedRoute.firstChild.firstChild.params, this.activatedRoute.queryParams,
-        (params, queryParams) => {
-          return { ...params, ...queryParams };
-        }).subscribe((params) => {
-          if (params.dialCode) {
-            const source = params.source || 'search';
-            applink = `${applink}&referrer=utm_source=${utm_source}&utm_medium=${source}&utm_campaign=dial&utm_term=${params.dialCode}`;
-          } else {
-            applink = `${applink}&referrer=utm_source=${utm_source}&utm_medium=get&utm_campaign=redirection`;
-          }
-          this.redirect(applink.replace(/\s+/g, ''));
-        });
+    const playstoreLink = _.get(this.tenantFooter, 'playstoreLink');
+    if (playstoreLink) {
+      // For iGot the URL is direclty taken; no UTM needed
+      this.redirect(playstoreLink);
     } else {
-      const path = this.router.url.split('/')[1];
-      applink = `${applink}&referrer=utm_source=${utm_source}&utm_medium=${path}`;
-      this.redirect(applink);
+      let applink = this.configService.appConfig.UrlLinks.downloadDikshaApp;
```

Comment:
Change the key name all the places where ever it has used,
downloadDikshaApp => downloadApp

Reply:
- This was an existing feature
- This should be changed at many occurrences
- Also, the URL for play store is hard coded in app.config.json; so the reference remains and cannot be changes